proposals/lip-define_new_transaction_schema.md

```diff
@@ -0,0 +1,263 @@
+```
+LIP: <lip number>
+Title: Define new transaction schema
+Author: Grigorios Koumoutsos <grigorios.koumoutsos@lightcurve.io>
+        Maxime Gagnebin <maxime.gagnebin@lightcurve.io>
+
+? Discussions-To: https://research.lisk.com/t/define-new-transaction-schema/348/15
+Type: Standards Track
+Created: <YYYY-MM-DD>
+Updated: <YYYY-MM-DD>
+```
+
+## Abstract
+
+This LIP defines a new schema to be used to serialize transactions. The main change is to replace module and command identifiers by the corresponding names, which are of type string. This LIP also updates the terminology used for transaction and transaction properties.
+
+
+## Copyright
+
+This LIP is licensed under the [Creative Commons Zero 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/).
+
+
+## Motivation
+
+The Lisk protocol handles identifiers for transactions, modules, commands, and many more. In many of those cases such as modules and commands there is also a name property, which is of type string and is set to some intuitive value (e.g., "token" module, "token transfer" command). In the current protocol, such objects are referenced using their identifiers and their names have an auxiliary role. This is not so convenient for users and developers, since they have to memorize the (non-intuitive) identifier values. Merging those two properties and identifying modules and commands using their name provides a much better user/developer experience.
```

Comment:
Just a small comment that command name example is `tokenTransfer` camelCase

Reply:
Good point, thanks! Fixed.